TestCLRT.py

```diff
@@ -0,0 +1,19 @@
+import sys
```

Comment:
Is having one test file a standard practice in python? In TDD workflow, we typically have one or more test files for each code file.